extension/solarwindsapmsettingsextension/extension.go

```diff
@@ -6,16 +6,26 @@ package solarwindsapmsettingsextension // import "github.com/open-telemetry/open
 import (
 	"context"
 	"crypto/tls"
+	"crypto/x509"
+	"encoding/binary"
+	"encoding/json"
+	"math"
+	"os"
 	"time"
 
-	"github.com/solarwindscloud/apm-proto/go/collectorpb"
+	"github.com/solarwinds/apm-proto/go/collectorpb"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/extension"
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 )
 
+const (
+	jsonOutputFile      = "/tmp/solarwinds-apm-settings.json"
```

Comment:
This does make the assumption that you're only going to be running on Unix based system, could I suggest that you use something like [os.TempDir](https://pkg.go.dev/os#TempDir) instead?

Reply:
For sure. We run on an Unix based system only. It doesn't hurt to use a cross platform function. 
Done.